pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2982,6 +2982,7 @@ func handlePhysicalIOAdapterListCreateModify(ctxArg interface{},
 		// Setup list first because functions lookup in IoBundleList
 		for _, phyAdapter := range phyIOAdapterList.AdapterList {
 			ib := *types.IoBundleFromPhyAdapter(phyAdapter)
+			// Preserve any existing IsPort/IsPCIBack/UsedByUUID
```

Comment:
nit.. Do we need this comment here? Seems more appropriate in AddOrUpdateIoBundle routine itself..

Reply:
It is actually stating an assumption in the caller. The fact that AddOrUpdate satisfies that assumption is a different matter. I can reword this with "We assume ..." if that makes sense.

Reply:
Done